README.md

```diff
@@ -157,6 +174,10 @@ rarefaction depth of 1500**
 **Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs**  
 `sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
 
+**Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs. Write the full output.**  
+`sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
```

Comment:
I think this command is missing ``--full_output`` (it's the same as the command above). 

Maybe for a future PR, but could we be more descriptive than ``--full_output``? It'd be nice if this parameter described what the output was (and I think that's more important than matching what ST1 did).